tests/internal/pack.c

```diff
@@ -241,6 +241,14 @@ void test_json_pack_mult_iter()
     flb_free(buf);
 }
 
+/* Validate default values of macros used in flb_msgpack_raw_to_json_sds */
+void test_msgpack_to_json_macros()
+{
+    /* Verify default values */
+    TEST_CHECK(FLB_MSGPACK_TO_JSON_INIT_BUFFER_SIZE == 2.0);
+    TEST_CHECK(FLB_MSGPACK_TO_JSON_REALLOC_BUFFER_SIZE == 0.10);
+}
+
```

Comment:
@edsiper 

Added this unit test to verify the default values. I believe this also verifies the values are type `float` and not strings. Is there another way to test this without having to `hard code` the default values ?